src/pages/_app.tsx

```diff
@@ -1,9 +1,24 @@
 import type { AppProps } from "next/app";
+import { NextUIProvider, globalCss } from "@nextui-org/react";
 
-import "@styles/globals.css";
+// import "@styles/globals.css";
```

Comment:
Is this required anymore? Or did you comment it out to be cleaned up later on?

Reply:
Yes, I would like to assess if we should use globalCss function or global.css for our global styles. Do share your opinion. For now I commented it out to preview that we can set global styles using Next UI